packages/client/src/app/home/home-page/home-page.component.ts

```diff
@@ -13,6 +13,7 @@ export class HomePageComponent implements OnInit {
 
   ngOnInit(): void {
     this.authService.$currentUser.subscribe((curUser) => {
+      curUser.createdAt = new Date(curUser.createdAt);
```

Comment:
These changes will not be needed when the migration to moleculer will be over as Dates will be automaticaly parsed through a Request Interceptor